app/src/main/java/ch/epfl/culturequest/ArtDescriptionDisplayActivity.java

```diff
@@ -259,4 +264,15 @@ private void postImage(String url, BasicArtDescription artwork, List<String> bad
             return null;
         });
     }
+
+    private void shareImage(Uri uri, BasicArtDescription description) {
+        Intent shareIntent = new Intent(Intent.ACTION_SEND);
```

Comment:
Does Intent.ACTION_SEND triggers some Android builtin method that allows to share it with other apps ?

Reply:
Yes it enables you to send a message, like in other apps

Reply:
ok great !